src/middleware/util.js

```diff
@@ -6,4 +6,13 @@ function asyncMiddleware (fn) {
   }
 }
 
-export { asyncMiddleware }
+function bearerToken (req, res, next) {
+  if (req.headers.authorization?.split(' ')[0] === 'Bearer') {
```

Comment:
Would `startsWith()` work here, instead of needing to split the array twice?

Reply:
Good catch! Even with startWith, we are doing an additional lookup. The best way is to just store the array. Fixed